float32[4] delta_x
float32[4] delta_y
float32[4] delta_z
float32[4] delta_yaw